group/init/v2/schema.js

```diff
@@ -19,16 +19,27 @@ module.exports = {
     groupKey: { $ref: '#/definitions/groupKey' },
     tangles: {
       type: 'object',
-      required: ['group', 'members'],
+      required: ['group', 'members', 'epoch'],
+      additionalProperties: false,
       properties: {
-        group: { $ref: '#/definitions/tangle/root' },
+        group: { $ref: '#/definitions/tangle/any' },
```

Comment:
I feel like this is generalising in a way which is not helpful/ safe. I suggest we split out:
- group/init-root
- group/init-epoch

or something, because otherwise validators built on this will let you make an incorrect group root message with no errors raised

Reply:
Both can still have the same `type` (and should) but lets find some way to distinguish

Reply:
I split them out into initRoot and initEpoch now